src/test/java/edu/hm/hafner/analysis/IssueAssert.java

```diff
@@ -0,0 +1,113 @@
+        return this;
+    }
+
+    @SuppressWarnings("NonBooleanMethodNameMayNotStartWithQuestion")
+    public IssueAssert hasPriority(final Priority priority) {
+        isNotNull();
+        if (!Objects.equals(actual.getPriority(), priority)) {
+            failWithMessage("Expected issue's priority to be <%s> but was <%s>", priority, actual.getPriority());
+        }
+        return this;
+    }
+
+    @SuppressWarnings("NonBooleanMethodNameMayNotStartWithQuestion")
+    public IssueAssert hasMessage(final String message) {
+        checkString("message", message, actual.getMessage());
+        return this;
+    }
+
+    @SuppressWarnings("NonBooleanMethodNameMayNotStartWithQuestion")
+    public IssueAssert hasDescription(final String description) {
+        checkString("description", description, actual.getDescription());
+        return this;
+    }
+
+    @SuppressWarnings("NonBooleanMethodNameMayNotStartWithQuestion")
+    public IssueAssert hasPackageName(final String packageName) {
+        checkString("packageName", packageName, actual.getPackageName());
+        return this;
+    }
+
+    @SuppressWarnings("NonBooleanMethodNameMayNotStartWithQuestion")
+    public IssueAssert hasFingerprint(final String fingerprint) {
+        checkString("fingerprint", fingerprint, actual.getFingerprint());
+        return this;
+    }
+
+    @SuppressWarnings("NonBooleanMethodNameMayNotStartWithQuestion")
+    public IssueAssert hasLineStart(final int lineStart) {
+        checkInt("lineStart", lineStart, actual.getLineStart());
+        return this;
+    }
+
+    @SuppressWarnings("NonBooleanMethodNameMayNotStartWithQuestion")
+    public IssueAssert hasLineEnd(final int lineEnd) {
+        checkInt("lineEnd", lineEnd, actual.getLineEnd());
+        return this;
+    }
+
+    @SuppressWarnings("NonBooleanMethodNameMayNotStartWithQuestion")
+    public IssueAssert hasColumnStart(final int columnStart) {
+        checkInt("columnStart", columnStart, actual.getColumnStart());
+        return this;
+    }
+
+    @SuppressWarnings("NonBooleanMethodNameMayNotStartWithQuestion")
+    public IssueAssert hasColumnEnd(final int columnEnd) {
+        checkInt("columnEnd", columnEnd, actual.getColumnEnd());
+        return this;
+    }
+
+    @SuppressWarnings("NonBooleanMethodNameMayNotStartWithQuestion")
+    private void checkString(final String usage, final String expected, final String actualString) {
+        isNotNull();
+        if (!Objects.equals(actualString, expected)) {
+            failWithMessage("Expected issue's %s to be <%s> but was <%s>", usage, expected, actualString);
+        }
+    }
+
+    @SuppressWarnings("NonBooleanMethodNameMayNotStartWithQuestion")
+    private void checkInt(final String usage, final int expected, final int actualInt) {
+        isNotNull();
+        if (actualInt != expected) {
+            failWithMessage("Expected issue's %s to be <%s> but was <%s>", usage, expected, actualInt);
```

Comment:
Der Standard ist:
`private static final String EXPECTED_BUT_WAS_MESSAGE = "%nExpecting:%n <%s>%nto be equal to:%n <%s>%nbut was not.";`